qiskit/primitives/containers/bindings_array.py

```diff
@@ -162,12 +132,7 @@ def kwvals(self) -> dict[tuple[str, ...], np.ndarray]:
     @property
     def num_parameters(self) -> int:
```

Comment:
Now that it's always kwvals, I wonder if we should add a `parameters` method similar to `QuantumCircuit.parameters`. Though im not sure what to do here, since in this class all `Parameter` inputs are unrwapped to their string `name`, but with actual `Parameter` objects you get the situation where two parameters with the same name are not always equal. Eg:
```
x = Parameter("a")
y = Parameter("a")
x == y  # False
```